include/boost/compute/iterator/strided_iterator.hpp

```diff
@@ -15,6 +15,7 @@
 #include <iterator>
 
 #include <boost/config.hpp>
+#include <boost/static_assert.hpp>
```

Comment:
Doesn't look like this is used here, just included for the test?

Reply:
Oh, sorry. I meant to include that in the test. I'll fix it in a moment.